### PR TITLE
enh: update applications-monitoring-poller

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: pp/integrations/plugin-packs/procedures/applications-monitoring-centreon-poller.md [hunk 1]
```diff
@@ -5,68 +5,87 @@ title: Centreon Poller
...
+<TabItem value="App-Monitoring-Centreon-Poller-custom" label="App-Monitoring-Centreon-Poller-custom">
 
-* App-Monitoring-Centreon-Poller-custom
+| Service Alias   | Service Template                                   | Service Description                        |
+|:----------------|:---------------------------------------------------|:-------------------------------------------|
+| Broker-Stats    | App-Monitoring-Centreon-Broker-Stats-Poller-custom | Check Centreon Broker processes statistics |
+| proc-centengine | App-Monitoring-Centreon-Process-centengine-custom  | Check centreon-engine process              |
+| proc-gorgoned   | App-Monitoring-Centreon-Process-gorgoned-custom    | Check centcore process                     |
+| proc-ntpd       | App-Monitoring-Centreon-Process-ntpd-custom        | Check NTP process                          |
+| proc-sshd       | App-Monitoring-Centreon-Process-sshd-custom        | Check sshd process                         |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-monitoring-centreon-poller.md:27]:
```suggestion
| proc-sshd       | App-Monitoring-Centreon-Process-sshd-custom        | Check SSHD process                         |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-poller.md [hunk 2]
```diff
@@ -5,68 +5,88 @@ title: Centreon Poller
...
 
-* App-Monitoring-Centreon-Poller-custom
+| Alias           | Modèle de service                                  | Description                                                                    |
+|:----------------|:---------------------------------------------------|:-------------------------------------------------------------------------------|
+| Broker-Stats    | App-Monitoring-Centreon-Broker-Stats-Poller-custom | Contrôle les statistiques des processus Centreon Broker                        |
+| proc-centengine | App-Monitoring-Centreon-Process-centengine-custom  | Contrôle permettant de vérifier le fonctionnement du processus centreon-engine |
+| proc-gorgoned   | App-Monitoring-Centreon-Process-gorgoned-custom    | Contrôle permettant de vérifier le fonctionnement du processus centcore        |
+| proc-ntpd       | App-Monitoring-Centreon-Process-ntpd-custom        | Contrôle permettant de vérifier le fonctionnement du processus NTP             |
+| proc-sshd       | App-Monitoring-Centreon-Process-sshd-custom        | Contrôle permettant de vérifier le fonctionnement du processus sshd            |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-poller.md:28]:
```suggestion
| proc-sshd       | App-Monitoring-Centreon-Process-sshd-custom        | Contrôle permettant de vérifier le fonctionnement du processus SSHD |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-poller.md [hunk 3]
```diff
@@ -101,101 +121,607 @@ ssh-copy-id -i ~/.ssh/id_ed25519.pub centreon@<IP_POLLER>
 
 ##
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-poller.md:122]: empty section?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-poller.md [hunk 4]
```diff
@@ -5,68 +5,88 @@ title: Centreon Poller
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-poller.md:7]:
```suggestion

Les collecteurs Centreon doivent être supervisés par le serveur central.

```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-poller.md [hunk 5]
```diff
@@ -75,23 +95,23 @@ Il apporte les Modèles de Service suivants :
...
 ### SNMP
 
-SNMP doit être configuré sur chaque collecteur supervisé. Vous pouvez vous aider de cette [documentation](operatingsystems-linux-snmp.md#prerequis) pour mettre en place rapidement une simple configuration SNMP. 
+SNMP doit être configuré sur chaque collecteur supervisé. Vous pouvez vous aider de [cette documentation](operatingsystems-linux-snmp.md#prerequis) pour mettre en place rapidement une configuration SNMP simple. 
 
-### Echange de clef SSH
+### Échange de clef SSH
 
-Un des services accompagnant le pack dispose d'une vérification réalisée à l'aide de SSH. Le serveur Central doit pouvoir se connecter à tous les collecteurs supervisés. 
+Un des services accompagnant le pack dispose d'une vérification réalisée à l'aide de SSH. Le serveur central doit pouvoir se connecter à tous les collecteurs supervisés.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-poller.md:102]: de quel service on parle? dans que cas on doit faire l'échange de clés?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-poller.md [hunk 6]
```diff
@@ -75,23 +95,23 @@ Il apporte les Modèles de Service suivants :
...
 
-1. Depuis le collecteur supervisé par le Central, paramétrer un mot de passe pour l'utilisateur **centreon** :
+1. Depuis le collecteur supervisé par le central, paramétrer un mot de passe pour l'utilisateur **centreon** :
 
 ```
 passwd centreon
 ```
 
-2. Depuis le Central, créer et copier la nouvelle clef SSH de l'utilisateur **centreon-engine** vers le collecteur : 
+2. Depuis le central, créer et copier la nouvelle clef SSH de l'utilisateur **centreon-engine** vers le collecteur :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-poller.md:114]: interaction entre les 2 utilisateurs?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/applications-monitoring-centreon-poller.md [hunk 7]
```diff
@@ -5,68 +5,89 @@ title: Centreon Poller
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+Centreon collectors must be supervised by the central server.
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-monitoring-centreon-poller.md:8]:
```suggestion
Centreon collectors must be monitored by the central server.
```

**Review by omercier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**